### PR TITLE
sys-apps/arch-chroot: new package, add 28

### PR DESCRIPTION
**Pull request by alfredfo**

### CODE REVIEW

File: sys-apps/arch-chroot/arch-chroot-28.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1
+
+DESCRIPTION="Generate output suitable for addition to an fstab file"
```
Comment (thesamesam) [inline sys-apps/arch-chroot/arch-chroot-28.ebuild:8]: fix

File: sys-apps/arch-chroot/arch-chroot-28.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+HOMEPAGE="https://github.com/archlinux/arch-install-scripts"
+SRC_URI="https://github.com/archlinux/arch-install-scripts/archive/refs/tags/v${PV}.tar.gz -> arch-install-scripts-v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=""
```
Comment (thesamesam) [inline sys-apps/arch-chroot/arch-chroot-28.ebuild:16]: drop RDEPEND+DEPEND

File: sys-apps/arch-chroot/arch-chroot-28.ebuild (hunk 1 above)
Comment (alfredfo) [reply sys-apps/arch-chroot/arch-chroot-28.ebuild:8]: oops

File: sys-apps/arch-chroot/arch-chroot-28.ebuild (hunk 2 above)
Comment (alfredfo) [reply sys-apps/arch-chroot/arch-chroot-28.ebuild]: done

Comment (alfredfo) [reply sys-apps/arch-chroot/arch-chroot-28.ebuild]: everything's good now I think

**Pull request closed without merging**